### DISCUSSION TITLE
How to use when there are multiple navigation controllers

### DISCUSSION THREAD

**Discussion by ibhavin** [LeoNatan/LNPopupController#451] [General] [answered]
Basically, I have sidemenu in my app. When I change the view from sidemenu it also changes its navigation controller. So, how can reinitialize popupcontroller?

**Comment by iDevelopper** [comment]
@ibhavin ,

What do you use for the side menu?

**Reply by ibhavin** [reply]
I am using [LGSideMenuController](https://github.com/Friend-LGA/LGSideMenuController) for sidemenu.

**Reply by LeoNatan** [reply]
@ibhavin Since it's a `UIViewController` subclass, try presenting on your side menu controller. See if that behaves as you expect.

**Reply by ibhavin** [reply]
I have tried it by presenting from sideMenuController, it working good for that navigation stack. But after selecting another view (which is from another navigation controller) from sidemenu, buttons from bottombar not responding.

**Reply by LeoNatan** [reply]
Hmm. Perhaps you should be presenting and dismissing the popup bar on each navigation controller. From a UX standpoint, I’m not sure what to recommend.

**Comment by LeoNatan** [comment, marked as answer]
You should probably present on the side menu controller directly. If it’s not a controller, it’s probably not a good framework to use.
